src/edu/csus/ecs/pc2/shadow/ShadowController.java

```diff
@@ -690,11 +692,33 @@ private Map<String, ShadowJudgementInfo> getJudgementsMap(Run[] pc2Runs) {
                         } else {
                             // we've exhausted methods of obtaining an acronym
                             log.warning("Null judgement acronym for run " + run.getNumber() + ", judgement string " + judgementString + "; skipping");
```

Comment:
Are we leaving the old message here indicating the entry is being "skipped", even though it might not be if the acronym replacement works?

Reply:
I agree with @johnbrvc :  I think the word "skipping" should be removed from this log message since we're no longer actually skipping the run (I think the log message should be retained; it just shouldn't say "skipping"...)